forge-gui/res/cardsfolder/upcoming/prized_statue.txt

```diff
@@ -0,0 +1,8 @@
+Name:Prized Statue
+ManaCost:2
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield or is put into a graveyard from the battlefield, create a Treasure token.
```

Comment:
please add `(It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")` to TriggerDescription

you can also add it to line 5 if you like (your call)